machine/lenovo/lenovo_ne10032/rootconf/sysroot-lib-onie/sysinfo-platform

```diff
@@ -0,0 +1,25 @@
+
+get_serial_num()
```

Comment:
you don't need this function, as this is the default behavior:

https://github.com/opencomputeproject/onie/blob/master/rootconf/grub-arch/sysroot-lib-onie/sysinfo-arch#L11-L21